web/src/app/details/DetailsPage.js

```diff
@@ -125,12 +132,20 @@ export default class DetailsPage extends React.PureComponent {
           <Card>
             <CardContent>
               {icon && <div className={classes.iconContainer}>{icon}</div>}
-              <Typography variant='h5'>{title}</Typography>
-              <Typography name='details' variant='subtitle1'>
+              <Typography
+                data-cy='details-heading'
+                className={classes.mainHeading}
+                variant='h2'
+              >
+                {title}
+              </Typography>
+              <Typography data-cy='details' variant='subtitle1'>
                 <Markdown value={details} />
               </Typography>
               {titleFooter && (
-                <Typography variant='subtitle1'>{titleFooter}</Typography>
+                <Typography variant='subtitle1' data-cy='time-zone'>
```

Comment:
I see that you're using `data-cy=time-zone` in a test for the schedules here, but since this is the main details component that can be reused in many places, I'd suggest we name it something a little more generic, such as `data-cy='title-footer'` or something